libgit/repo.go

```diff
@@ -238,6 +244,7 @@ func UpdateRepoMD(ctx context.Context, config libkbfs.Config,
 			RefName:              string(refName),
 			Commits:              kbCommits,
 			MoreCommitsAvailable: hasMoreCommits,
+			IsDelete:             refData.IsDelete,
```

Comment:
If there are no commits for this ref (and there shouldn't be for deleted refs, right?)  then this metadata won't ever be appended, so it seems like the delete wouldn't get sent at all, right?

Reply:
I don't follow...we're iterating over `refData` here, and we haven't used `range refData.Commits` to skip out of this completely. A ref with no commits will still send an `UpdateRepoMD` (as indeed we do with repo creations).

Reply:
Oh sorry, it looked like this was in the `range refData.Commits` loop.  I misread the github tabbing.